ddtrace/bootstrap/sitecustomize.py

```diff
@@ -262,6 +259,7 @@ def _(threading):
 
         enable_appsec_rc()
 
+    config._ddtrace_bootstrapped = True
```

Comment:
is this accurate? should we instead emit an env var or something from `ddtrace/commands/ddtrace_run.py` ?

since `ddtrace/auto.py` just imports `ddtrace.bootstrap.sitecustomize`

Reply:
I don't think it matters if `ddtrace-run` is used or `ddtrace.bootstrap.sitecustomize` or `ddtrace.auto` is imported. I just want to track if this file is executed.

Reply:
I considered naming the telemetry configuration `ddtrace_was_boostraped` but that name feels a bit cryptic.

Reply:
🤔 I _think_ we do want to know the difference?

since with `ddtrace-run` we are controlling imports and patching for a specific ordering, and with `import ddtrace.auto` they can still import that in an incorrect location.... ?

checking `sys.modules` could be "interesting" approach (not convinced this is a good idea, was just curious what it could look like)

```python
import sys


ddtrace_auto_used = "ddtrace.auto" in sys.modules
```

but with `ddtrace-run` we don't get `ddtrace.bootstrap.sitecustomize` since we change the path, so it gets imported as `sitecustomize`, but it could be any `sitecustomize`, not necessarily ours.


(these are random musings, not a request for a change)

Reply:
Yup, we need to check sys.path to get if `ddtrace_run` was used.

Reply:
At least for otel it does not matter. Both will configure the tracerprovider. But in general distinguishing between the two can help us resolve escalations.

Maybe we should send another configuration, `ddtrace_auto_used`.